### PR TITLE
Commands support for .fld.gz

### PR DESCRIPTION
**Pull request by Henrybk**
Since we are now using .fld.gz files we should change these commands
also.

### CODE REVIEW

File: src/Commands.pm [hunk 1]
```diff
@@ -3287,7 +3287,7 @@ sub cmdMove {
 				# map
 				$map_or_portal =~ s/^(\w{3})?(\d@.*)/$2/; # remove instance. is it possible to move to an instance? if not, we could throw an error here
 				# TODO: implement Field::sourceName function here once they are implemented there - 2013.11.26
-				my $file = $map_or_portal.'.fld';
+				my $file = $map_or_portal.'.fld.gz';
 				$file = File::Spec->catfile($Settings::fields_folder, $file) if ($Settings::fields_folder);
 				$file .= ".gz" if (! -f $file); # compressed file
```
Comment (allanon) [inline src/Commands.pm]: Doesn't this line already support `.fld.gz`? We need to keep compatibility with non-gzipped `.fld` files too.

File: src/Commands.pm [hunk 2]
```diff
@@ -4025,9 +4025,9 @@ sub cmdPortalList {
 		#Command: portals add y_airport 143 43 y_airport 148 51 0 c r0 c r0
 		print $args."TEST\n";
 		my ($srcMap, $srcX, $srcY, $dstMap, $dstX, $dstY, $seq) = $args =~ /^add ([a-zA-Z\_\-0-9]*) (\d{1,3}) (\d{1,3}) ([a-zA-Z\_\-0-9]*) (\d{1,3}) (\d{1,3})(.*)$/; #CHECKING
-		my $srcfile = $srcMap.'.fld';
+		my $srcfile = $srcMap.'.fld.gz';
```
Comment (allanon) [inline src/Commands.pm]: Should write compatibility code so the old `.fld` will still work. See line 3292 above for an example.

File: src/Commands.pm (hunk 1 above)
Comment (Henrybk) [inline src/Commands.pm]: I tested it with 'portals add' and it won't work

File: src/Commands.pm [hunk 3]
```diff
@@ -5989,4 +5991,4 @@ sub cmdDeadTime {
 	message $msg, "list";
 }
 
-1;
\ No newline at end of file
+1;
```
Comment (allanon) [inline src/Commands.pm]: :heart:

**Comment by allanon** [comment]
:+1:

**Comment by ya4ept** [comment]
you can do it in a single commit, not three?
Please, applying this [patch](https://gist.github.com/ya4ept/559965eebd0262a598ad) additionally

**Pull request merged**